Frontend/src/components/RichTextEditorFormContol/RichTextEditorFormControl.tsx

```diff
@@ -46,7 +46,11 @@ export class RichTextEditorFormControl extends React.Component<
             height: this.props.height || "calc(var(--spacing) * 20)",
             overflow: "scroll",
             borderBottom: "1px solid var(--c-grey-d20)"
-          }
+          },
+          placeHolder: {
+            // make the whole editor clickable
+            height: "calc(100% - 55px)" 
```

Comment:
What is the reason for this 55px?

Reply:
It's determined inside the mui-rte component; I couldn't find the exact rule... so I decided to put the exact value in here.